src/applications/personalization/profile-2/components/ProfileWrapper.jsx

```diff
@@ -63,12 +63,34 @@ class ProfileWrapper extends Component {
   // note that `children` will be passed in via React Router.
   mainContent = () => (
     <>
+      {/* the mobile sidenav trigger button */}
+      <button
+        type="button"
+        className="va-btn-sidebarnav-trigger"
+        aria-controls="va-detailpage-sidebar"
+      >
+        <span>
+          <b>Profile Menu</b>
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
```

Comment:
Can we import this svg rather than inlining it?

Reply:
We could! This was a copy/paste job

Reply:
Do you mind if I wait to do this until I make the mobile menu trigger its own component? Should be part of my next PR for this work.